### PR TITLE
Added tiny examples to every function

### PR DESCRIPTION
**Pull request by namsaraeva**
Added examples to some functions

### CODE REVIEW

File: ehrapy/io/_write.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def write(
     else:
         # dummy encoding when there is non numerical data in X
         if not np.issubdtype(adata.X.dtype, np.number) and extension == "h5ad":
-            # flag to indicate an Anndata object has been dummy encoded to write it to .h5ad file
+            # flag to indicate an anndata object has been dummy encoded to write it to .h5ad file
```
Comment (Zethson) [inline ehrapy/io/_write.py:62]: Why this?

**Review by Zethson — COMMENTED** [review]
Almost there! Great and you also fixed a few old docstrings

File: ehrapy/io/_write.py (hunk 1 above)
Comment (namsaraeva) [reply ehrapy/io/_write.py:62]: Oh, was not really intended. We can rollback

Comment (Zethson) [reply ehrapy/io/_write.py:62]: yes pls

then merge it :)

**Pull request merged**